Libraries/src/Amazon.Lambda.RuntimeSupport/Bootstrap/UserCodeLoader.cs

```diff
@@ -406,5 +428,35 @@ private object GetCustomerObject(Type customerType)
 
             return Activator.CreateInstance(customerType);
         }
+
+        private MethodInfo GetCustomerInitializer(Type customerType)
+        {
+            // TODO Should the method require an environment variable to opt-in/out?
```

Comment:
I'm very torn on this. The environment variable would avoid any potential conflict if somebody already has an `InitializeAsync` plus you could configure your own name. But the discoverability would be really low.

Wonder if we made the name more unique like `LambdaInitializeAsync` that would avoid the possible naming collision.

@ganeshnj Do you have any thoughts on this?

Reply:
There has to a way to avoid calling any customer method that might have same signature. I can't think of anything other than env var as of now.

I would prefer `InitializeLambdaAsync` or `LambdaInitializeAsync` which provide better context that this method not meant to be called in customer code (customer can if they want but not recommended) and will be executed as part of init phase.